CHANGES.md

```diff
@@ -11,6 +11,13 @@ unreleased
 
 - Format `dune printenv` output (#1867, fix #1862, @emillon)
 
+1.7.3 (unreleased)
+------------------
+
+- Fix interpretation of `META` files containing archives with `/` in
+  the filename. This was causing llvm to be unusable with dune (#1889,
```

Comment:
It might be more helpful to people reading the changelog if a less specific description was given. I don't think that llvm is so special, wouldn't the same issue arise for any lib with sublibs?

Reply:
Before dune, all sublibs were usually installed in the main directory since that was the easiest thing to do. Sublibs installed in subdirs generally have a `directory` field (that's what dune does for instance) and since filenames are relative to that directory, filenames still don't contain slashes.

So llvm is probably one of a very small set of projects to be affected, so it seems worth mentioning to me. I added "for instance" to make it clear it's just an example

Reply:
I see, thanks for the explanation. (Maybe there is too much flexibility somewhere...)